src/PriceCalculatorKata/Tax.cs

```diff
@@ -0,0 +1,10 @@
+namespace PriceCalculatorKata;
+public class Tax
+{
+    public static float UniversalTax { get; set; }
+
+    public static double CalculateTaxValue(double price)
```

Comment:
this shouldn't be static. you love static a lot :) this can be changes through the app, as any other property.

Reply:
I think maybee you just need a service for taxes, and pass the universal tax as a parameter to the method? instead of saving it as a prop?